eventuate-tram-consumer-jdbc-micronaut/src/main/java/io/eventuate/tram/consumer/jdbc/micronaut/EventuateMicronautTransactionalNoopDuplicateMessageDetector.java

```diff
@@ -0,0 +1,24 @@
+package io.eventuate.tram.consumer.jdbc.micronaut;
+
+import io.eventuate.common.jdbc.micronaut.EventuateMicronautTransactionManagement;
+import io.eventuate.tram.consumer.common.DuplicateMessageDetector;
+import io.eventuate.tram.consumer.common.SubscriberIdAndMessage;
+
+public class EventuateMicronautTransactionalNoopDuplicateMessageDetector implements DuplicateMessageDetector {
+
+  private EventuateMicronautTransactionManagement eventuateMicronautTransactionManagement;
+
+  public EventuateMicronautTransactionalNoopDuplicateMessageDetector(EventuateMicronautTransactionManagement eventuateMicronautTransactionManagement) {
+    this.eventuateMicronautTransactionManagement = eventuateMicronautTransactionManagement;
+  }
+
+  @Override
+  public boolean isDuplicate(String consumerId, String messageId) {
+    return false;
+  }
+
+  @Override
+  public void doWithMessage(SubscriberIdAndMessage subscriberIdAndMessage, Runnable callback) {
+    eventuateMicronautTransactionManagement.doWithTransaction(connection -> callback.run());
```

Comment:
This isn't right. doWithMessage() should simply call callback.run()

Reply:
But original version uses transactions